td.vue/src/i18n/en.js

```diff
@@ -66,6 +66,10 @@ const en = {
         diagrams: 'Diagrams',
         addNewDiagram: 'Add a new diagram...',
         threats: 'Threats',
+        errors: {
+            save: 'Error saving Threat Model.  Please check the developer console for more information'
```

Comment:
Very small nit, but there are 2-3 spaces after the full stop.

Reply:
I wanted to double-check before commenting: Everything containing a period has a double-space at the end.  It's how I was taught... you know, back when dinosaurs were around.  I'll update all of them. :grin: